src/Main.java

```diff
@@ -1,5 +1,92 @@
+import java.util.Formatter;
+
 public class Main {
+    private static Formatter Arrays;
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+
+        int[] weight = new int[]{1, 2, 3};
```

Comment:
этот способ создания массива аналогичен такому
```suggestion
        int[] weight = {1, 2, 3};
```
а требовался вот такой
```suggestion
        int[] weight = new int[3];
        weight[0] = 1;
        weight[1] = 2;
        weight[2] = 3;
```